src/components/hero/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import { Koros, Button } from "hds-react";
+import Link from "next/link";
+
+import { LandingPage } from "../../types";
+import styles from "./hero.module.scss";
+
+function Hero({ title, link, desktopImage }: LandingPage) {
+  return (
+    <div
+      className={styles.hero}
+      style={{ backgroundImage: `url(${desktopImage.uri})` }}
+    >
+      <div className={styles.box}>
+        <p>Kokosimme yhteen</p>
+        <h1>{title}</h1>
+        <Link href={link}>
+          <Button className={styles.button}>Katso vinkit</Button>
```

Comment:
Would it be more appropriate to use an anchor element?

Reply:
I guess so, might require a little more styling.